helm-hooks/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN make ${stage}
 FROM golang:1.12
 RUN go get -u github.com/cloudflare/cfssl/cmd/...
 
-ENV KUBECTL_VERSION v1.10.0
+ENV KUBECTL_VERSION v1.16.4
```

Comment:
[note]
Use the same version as grafeas/kritis:
https://github.com/grafeas/kritis/blob/913f0bddcbb2777f7bb5ab24b373e5a490bea694/helm-hooks/Dockerfile#L25